linuxkm/linuxkm_wc_port.h

```diff
@@ -946,14 +1085,14 @@
         _alloc_sz;                                                         \
     })
     #ifdef HAVE_KVMALLOC
-        #define malloc(size) kvmalloc_node(WC_LINUXKM_ROUND_UP_P_OF_2(size), GFP_KERNEL, NUMA_NO_NODE)
+        #define malloc(size) kvmalloc_node(WC_LINUXKM_ROUND_UP_P_OF_2(size), GFP_ATOMIC, NUMA_NO_NODE)
```

Comment:
Could be disadvantages to blanket switching all to GFP_ATOMIC?

Reply:
Indeed!

(Fixed as discussed.)